packages/serverless/src/domain/use-case/user.ts

```diff
@@ -1,21 +1,38 @@
 import Client from 'mina-signer';
-import { ClientSession } from 'mongodb';
+import { ClientSession, FindOptions } from 'mongodb';
+import logger from '../../helper/logger.js';
 import ModelUser from '../../model/global/user.js';
 import { Pagination } from '../types/pagination.js';
 import { Signature } from '../types/proof.js';
 import { User } from '../types/user.js';
-import { FilterCriteria, parseQuery } from '../utils/document.js';
-import logger from '../../helper/logger.js';
+import { FilterCriteria } from '../utils/document.js';
+import { objectToLookupPattern } from '../../helper/common.js';
 
-// eslint-disable-next-line import/prefer-default-export
+export async function searchUser(
+  query: { [key: string]: string },
+  pagination?: Pagination
```

Comment:
Can you handle the case when pagination is undefined? We should have the default value

Reply:
well I just put a pagination into options, if undefined it will be select all